QgisModelBaker/gui/options.py

```diff
@@ -203,3 +204,26 @@ def keyPressEvent(self, e):
             _selected_indexes = self.selectedIndexes()
             self.space_pressed.emit(_selected_indexes[0])
         super(ModelListView, self).keyPressEvent(e)
+
+
+class FileDropListView(QListView):
+
+    files_dropped = pyqtSignal(list)
+
+    def __init__(self, parent=None):
+        super(FileDropListView, self).__init__(parent)
+        self.setAcceptDrops(True)
+        self.setDragDropMode(QListView.InternalMove)
+
+    def dragEnterEvent(self, event):
+        event.acceptProposedAction()
```

Comment:
Should not this be accepted only if entered files are `in ["xtf", "XTF", "itf", "ITF", "ili"]` ?

Reply:
Yes, but if you would reject the event, the mouse cursor changes and let the user now that his files are not droppable in the widget.